File: backends/php-laravel/app/UseCases/Notification/Dtos/NotificationCreateDto.php
<?php

/**
 * This is a program developed by BobTabo.
 *
 * Copyright (c) 2026 BobTabo. All Rights Reserved.
 */

declare(strict_types=1);

namespace App\UseCases\Notification\Dtos;

use App\Support\Dtos\AbstractDto;

/**
 * 通知登録Dtoクラスです。
 *
 * @author Satoshi Nagashiba <satoshi.nagashiba@gmail.com>
 * @package App\UseCases\Notification\Dtos
 */
class NotificationCreateDto extends AbstractDto
{
    public ?int $messageType = null;
    public ?string $title = null;
    public ?string $message = null;
    public ?string $url = null;
}

File: backends/java-spring/src/main/java/com/authorization/usecases/notification/dtos/NotificationCreateDto.java
/**
 * This is a program developed by BobTabo.
 *
 * Copyright (c) 2026 BobTabo. All Rights Reserved.
 */
package com.authorization.usecases.notification.dtos;

import com.authorization.support.dtos.AbstractDto;
import lombok.Getter;
import lombok.Setter;

/**
 * 通知登録DTOクラスです。
 *
 * @author Satoshi Nagashiba <satoshi.nagashiba@gmail.com>
 */
@Getter
@Setter
public class NotificationCreateDto extends AbstractDto {

    private Integer messageType;
    private String title;
    private String message;
    private String url;
}
